Assets/Scripts/Buildings/Place.cs

```diff
@@ -154,6 +154,8 @@ private void LeftClick()
             Selected = Deselected;
 
             OnBuildingPlaced?.Invoke();
+            
+            GameManager.Manager.Achievements.Unlock(Achievement.BuildOneBuilding);
```

Comment:
I know this is early on, but I'll just flag that ideally we'd like to invert this logic, so that instead, each achievement would subscribe to an event, such as the `OnBuildingPlaced` callback here. Saves us from having achievement code popping up right in the middle of building placement and encapsulates concerns a lot better.